### PR TITLE
Adds CC-BY-ND-4.0 to schema

### PR DESCRIPTION
**Pull request by MoreRobustThanYou**

### CODE REVIEW

**Comment by politas** [comment]
So it's ok for people to make money off distributing your work, but not to change it? That's an interesting combination. I can see why we haven't previously included it.

File: CKAN.schema [hunk 1]
```diff
@@ -350,7 +350,7 @@
                 "CC-BY-NC", "CC-BY-NC-1.0", "CC-BY-NC-2.0", "CC-BY-NC-2.5", "CC-BY-NC-3.0", "CC-BY-NC-4.0",
                 "CC-BY-NC-SA", "CC-BY-NC-SA-1.0", "CC-BY-NC-SA-2.0", "CC-BY-NC-SA-2.5", "CC-BY-NC-SA-3.0", "CC-BY-NC-SA-4.0",
                 "CC-BY-NC-ND", "CC-BY-NC-ND-1.0", "CC-BY-NC-ND-2.0", "CC-BY-NC-ND-2.5", "CC-BY-NC-ND-3.0", "CC-BY-NC-ND-4.0",
-                "CC0",
+                "CC0", "CC-BY-ND-4.0",
```
Comment (politas) [inline CKAN.schema]: Please insert a complete set as per the preceding licence versions for each version and an unspecified version

    "CC-BY-ND", "CC-BY-ND-1.0", "CC-BY-ND-2.0", "CC-BY-ND-3.0", "CC-BY-ND-4.0",

And then leave CC0 on its own line as before.

**Comment by MoreRobustThanYou** [comment]
TBH I only added it to make adding ksprc textures for Spectra easier

File: CKAN.schema [hunk 2]
```diff
@@ -350,6 +350,7 @@
                 "CC-BY-NC", "CC-BY-NC-1.0", "CC-BY-NC-2.0", "CC-BY-NC-2.5", "CC-BY-NC-3.0", "CC-BY-NC-4.0",
                 "CC-BY-NC-SA", "CC-BY-NC-SA-1.0", "CC-BY-NC-SA-2.0", "CC-BY-NC-SA-2.5", "CC-BY-NC-SA-3.0", "CC-BY-NC-SA-4.0",
                 "CC-BY-NC-ND", "CC-BY-NC-ND-1.0", "CC-BY-NC-ND-2.0", "CC-BY-NC-ND-2.5", "CC-BY-NC-ND-3.0", "CC-BY-NC-ND-4.0",
+                "CC-BY-ND", "CC-BY-ND-1.0", "CC-BY-ND-2.0", "CC-BY-ND-3.0", "CC-BY-ND-4.0",
```
Comment (Olympic1) [inline CKAN.schema]: `"CC-BY-ND-2.5"` also exists

File: CKAN.schema [hunk 3]
```diff
@@ -350,6 +350,7 @@
                 "CC-BY-NC", "CC-BY-NC-1.0", "CC-BY-NC-2.0", "CC-BY-NC-2.5", "CC-BY-NC-3.0", "CC-BY-NC-4.0",
                 "CC-BY-NC-SA", "CC-BY-NC-SA-1.0", "CC-BY-NC-SA-2.0", "CC-BY-NC-SA-2.5", "CC-BY-NC-SA-3.0", "CC-BY-NC-SA-4.0",
                 "CC-BY-NC-ND", "CC-BY-NC-ND-1.0", "CC-BY-NC-ND-2.0", "CC-BY-NC-ND-2.5", "CC-BY-NC-ND-3.0", "CC-BY-NC-ND-4.0",
+                "CC-BY-ND", "CC-BY-ND-1.0", "CC-BY-ND-1.5", "CC-BY-ND-2.0", "CC-BY-ND-3.0", "CC-BY-ND-4.0",
```
Comment (Olympic1) [inline CKAN.schema]: Not 1.5 but **2**.5, like the ones above it

File: CKAN.schema (hunk 2 above)
Comment (politas) [reply CKAN.schema]: My bad, I missed that one.

**Pull request merged**